grawkit

```diff
@@ -148,13 +148,24 @@ function render_merge(idx, _, buf, tmp, refs, fields, m, i, hspc) {
 	for (i in fields) {
 		split(fields[i], m, "|")
 
-		# Add starting point to previous branch ending point.
-		tmp = "M" m[1] * style["branch/spacing"] "," m[2] * style["commit/spacing"]
-
-		# Add Bezier curve leading to current branch starting point.
-		tmp = tmp " C" hspc "," m[2] * style["commit/spacing"]
-		tmp = tmp " " hspc "," m[2] * style["commit/spacing"]
-		tmp = tmp " " hspc "," refs[1] * style["commit/spacing"]
+		if(length(m) == 2) {
```

Comment:
My memory is a bit fuzzy here, but when is `m` not equal to 2? My impression from looking at the code is that there should always be two sides to a merge, the origin and the destination. Is this not always the case?

Reply:
Please run the following test case (created on purpose to highlight the issue) in both versions and notice the difference:


git commit
git branch A
git branch B
git checkout A
git commit
git checkout B
git commit
git commit
git commit
git commit
git commit
git commit
git commit
git commit
git commit
git commit
git commit
git checkout A
git merge B
git checkout B
git commit
git commit
git checkout master
git merge B